db/migrate/20200327164455_add_mark_remove_column_to_recents.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class AddMarkRemoveColumnToRecents < ActiveRecord::Migration
+  disable_ddl_transaction!
+  def up
+    add_column :recents, :mark_remove, :boolean
+    change_column_default :recents, :mark_remove, false
+    remove_foreign_key :recents, :classifications
+
+    remove_old_recents
```

Comment:
Should we split the DDL and data work into 2 migrations? Does this mean we can avoid `disable_ddl_transaction!` for DDL steps?